core/modules/util/StringToVector.h

```diff
@@ -0,0 +1,51 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2020 LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+
+#ifndef LSST_QSERV_UTIL_STRINGTOVECTOR_H
+#define LSST_QSERV_UTIL_STRINGTOVECTOR_H
+
+// System headers
+#include <string>
+#include <vector>
+
+// Qserv headers
+#include "util/Command.h"
+
+namespace lsst {
+namespace qserv {
+namespace util {
+
+/// @return a vector of strings resulting from splitting 'str' into separate strings
+/// using 'separator' as the delimiter.
+std::vector<std::string> splitString(std::string const& str, std::string const& separator);
```

Comment:
I would argue that this and other method was put into a utility class rather than having them as stand-alone functions in the namespace.